src/components/upvote/container/upvoteContainer.js

```diff
@@ -53,7 +53,12 @@ class UpvoteContainer extends PureComponent {
     } = this.props;
 
     const author = get(content, 'author');
-    const isVoted = get(content, 'is_voted');
+    const isVoted = get(content, 'is_voted')
```

Comment:
in short case;

`const isVoted = get(content, 'is_voted', false)
      && parseInt(get(content, 'is_voted'), 10) / 10000;`

Reply:
this can be wrong line?